src/Theme.elm

```diff
@@ -40,19 +42,23 @@ lightOrange =
 -- Accent colours
 
 
-green : Color
+green : { colour : Color, string : String }
```

Comment:
might make sense to alias this record for re-use but I guess we only use it 3 times?

Reply:
I considered doing all of them, but that seemed a waste of time tracking down all the changes.

Reply:
Yeah - I think fine to go like this.